_pages/home.md

```diff
@@ -8,81 +8,63 @@ header:
+            <td>8&nbsp;January 2023</td>
+        </tr>
+        <tr>
+            <td>Author response period</td>
+            <td>Tuesday&ndash;Monday</td>
+            <td>6&ndash;12&nbsp;December 2022</td>
+        </tr>
+        <tr>
+            <td>Notification of acceptance <br/> (<i>long & short papers</i>)</td>
+            <td>Friday</td>
+            <td>20&nbsp;January 2023</td>
+        </tr>
+        <tr>
+            <td>Camera-ready papers due <br/> (<i>long & short papers</i>)</td>
+            <td>Fiday</td>
+            <td>10&nbsp;February 2023</td>
+        </tr>
+        <tr>
+            <td>Workshops & Tutorials</td>
+            <td>Tuesday and Saturday</td>
+            <td>2 and 6&nbsp;May 2023</td>
+        </tr>
         <tr>
-            <td>Main conference</td>
-            <td>Wednesday&ndash;Friday</td>
-            <td>21&ndash;23&nbsp;April, 2021</td>
+            <td>Main Conference</td>
+            <td>Wednesday &ndash; Friday</td>
+            <td>3&ndash;5&nbsp;May 2023</td>
         </tr>
     </tbody>
 </table>
 
 ##### All deadlines are 11.59 pm [UTC -12h](https://www.timeanddate.com/time/zone/timezone/utc-12) ("anywhere on Earth").
 
-## EACL 2021 Anti-Harassment Policy
+## EACL 2023 Anti-Harassment Policy
 
-EACL2021 adheres to the [ACL's code of ethics](https://www.aclweb.org/portal/content/acl-code-ethics), [ACL's anti-harassment policy](https://www.aclweb.org/adminwiki/index.php?title=Anti-Harassment_Policy), and [ACL’s code of conduct]({{ "/code" | relative_url }}). Violations of the code of ethics as well as complaints raised under the anti-harassment policy should be brought to the [Professional Conduct Committee](https://www.aclweb.org/adminwiki/index.php?title=Professional_Conduct_Committee), who can be reached during the conference via the PCC channel in RocketChat, or you may also contact any current member of the [EACL Executive Board](http://eacl.org/general/#officers). Please be assured that if you approach the PCC or EACL board members, your concerns will be kept in strict confidence, and we will consult with you on any actions taken.
+EACL2023 adheres to the [ACL's code of ethics](https://www.aclweb.org/portal/content/acl-code-ethics), [ACL's anti-harassment policy](https://www.aclweb.org/adminwiki/index.php?title=Anti-Harassment_Policy), and [ACL’s code of conduct]({{ "/code" | relative_url }}). Violations of the code of ethics as well as complaints raised under the anti-harassment policy should be brought to the [Professional Conduct Committee](https://www.aclweb.org/adminwiki/index.php?title=Professional_Conduct_Committee), who can be reached during the conference via the PCC channel in RocketChat, or you may also contact any current member of the [EACL Executive Board](http://eacl.org/general/#officers). Please be assured that if you approach the PCC or EACL board members, your concerns will be kept in strict confidence, and we will consult with you on any actions taken.
```

Comment:
The link "ACL's code of conduct" is broken. You probably want to add `code.md` back?

Reply:
Thanks, I added the page also to the navigation.